doc/connectivity/bluetooth/api/mesh/access.rst

```diff
@@ -190,6 +190,7 @@ Composition Data Page 1 provides information about the relationships between mod
 and is mandatory for all mesh devices. A model may extend and/or correspond to one
 or more models. A model can extend another model by calling :c:func:`bt_mesh_model_extend`,
 or correspond to another model by calling :c:func:`bt_mesh_model_correspond`.
+This API is enabled by the :kconfig:option:`CONFIG_BT_MESH_MODEL_EXTENSIONS` Kconfig option.
```

Comment:
This doc change, conflicts with PR commit message, and the KConfig change.

Reply:
Could you explain what conflict you see? This line is added to explain that `bt_mesh_model_extend` and `bt_mesh_model_correspond` are enabled by `CONFIG_BT_MESH_MODEL_EXTENSIONS`. This was not needed before because CDP1 was depending on `CONFIG_BT_MESH_MODEL_EXTENSIONS`. Now this dependency is removed, that is why I added this sentence.

Reply:
Well, I think this is fine. But technically the user can override the default 0 thing. It should not be a big issue though.

Reply:
I mean, a prj.conf could be:

BT_MESH_MODEL_EXTENSIONS=n
BT_MESH_MODEL_EXTENSION_LIST_SIZE=10

Reply:
> BT_MESH_MODEL_EXTENSIONS=n
BT_MESH_MODEL_EXTENSION_LIST_SIZE=10

Right, but `BT_MESH_MODEL_EXTENSIONS` adds extra 4 bytes for model structure. But a model can still use only `bt_mesh_model_correspond` which doesn't depend on `BT_MESH_MODEL_EXTENSIONS`. So this construction technically correct if models on a node only correspond to each other, but not extend other models.
https://github.com/zephyrproject-rtos/zephyr/blob/5b4c927c50eb032730767808b0304bf1043e2e91/include/zephyr/bluetooth/mesh/access.h#L895-L898
I removed this line to remove confusion. It is mentioned above on this page about `BT_MESH_MODEL_EXTENSIONS`.

Reply:
Now I get it. Good to remove that line from PR. The problem is with the word "this" in the PR description. I read it as if it referred to CDP1, but it actually refers to Model Extensions. 

Less confusing way to phrase this would be:

> The dependency on BT_MESH_MODEL_EXTENSIONS is removed because if a node contains only base models (for example Generic OnOff Server), the extensions API will consume extra RAM bytes which won't be used.